src/io/flutter/preview/PreviewView.java

```diff
@@ -416,6 +450,36 @@ private void scrollTreeToNodeOnTop(TreeNode node) {
     }
   }
 
+  private class QuickAssistAction extends AnAction {
+    QuickAssistAction(Icon icon, String assistMessage) {
+      super(icon);
+      messageToActionMap.put(assistMessage, this);
+    }
+
+    @Override
+    public void actionPerformed(AnActionEvent e) {
+      final SourceChange change;
+      synchronized (actionToChangeMap) {
+        change = actionToChangeMap.get(this);
+      }
+      if (change != null) {
+        ApplicationManager.getApplication().runWriteAction(() -> {
+          try {
+            AssistUtils.applySourceChange(project, change, false);
+          }
+          catch (DartSourceEditException ignored) {
```

Comment:
Should we provide feedback here? A toast? `FlutterMessages.showWarning()` may be useful.